custom_components/audiconnect/sensor.py

```diff
@@ -50,3 +50,13 @@ def device_class(self):
         ):
             return self._instrument.device_class
         return None
+
+    @property
+    def state_class(self):
+        """Return the state_class"""
+        if (
+            self._instrument.state_class is not None
```

Comment:
I don't think this is needed anymore.

Reply:
Was torn on removing it or not since it's now a benign final check. I can remove it tomorrow

Reply:
removed this and the device class check